tailwind.config.js

```diff
@@ -26,4 +26,8 @@ module.exports = {
     backgroundColor: ["responsive", "hover", "focus", "active"],
   },
   plugins: [require("@tailwindcss/ui"), require("tailwindcss-hyphens")],
+  purge: {
+    enabled: true,
+    content: ['./components/**/*.tsx', './pages/**/*.tsx'],
+  },
```

Comment:
```suggestion
  purge: ['./components/**/*.tsx', './pages/**/*.tsx'],
```
The `enabled: true` forcefully enables purging, not just during production (https://tailwindcss.com/docs/controlling-file-size#enabling-manually). We only want to use it for production though, because it makes the dev loop slower to purge on every build.

Reply:
Resolved on this commit https://github.com/denoland/deno_website2/pull/801/commits/4f316af543b1bbcd04ae451c6938e72c8ac5daa5